src/main/java/uk/gov/hmcts/reform/sendletter/tasks/UploadLettersTask.java

```diff
@@ -21,19 +22,19 @@
 
 import static java.time.LocalDateTime.now;
 
+@Component
 public class UploadLettersTask {
     private static final Logger logger = LoggerFactory.getLogger(UploadLettersTask.class);
 
     public static String SMOKE_TEST_LETTER_TYPE = "smoke_test";
 
     private final LetterRepository repo;
-    private final Zipper zipper;
+    private final Zipper zipper = new Zipper();
     private final FtpClient ftp;
 
     @Autowired
-    public UploadLettersTask(LetterRepository repo, Zipper zipper, FtpClient ftp) {
+    public UploadLettersTask(LetterRepository repo, FtpClient ftp) {
```

Comment:
why?

Reply:
The Zipper was causing the Application context to fail to initialise (though that may have been fixed).

The Zipper isn't currently mocked/asserted on in any tests and doesn't have any state, so we could replace it with static method calls?

Reply:
For example why is the FileNameHelper static methods but the Zipper an instance class?

Reply:
will fix